src/main.cc

```diff
@@ -0,0 +1,79 @@
+#include <bits/stdc++.h>
```

Comment:
Include what you see.
https://www.gangofcoders.net/solution/why-should-i-not-include-bits-stdc-h/